### PR TITLE
Optimize Dockerfile

### PR DESCRIPTION
**Pull request by ghorbani-ali**
Replace SDK base image with alpine to lightweight build stage image.
Use dynamic package build for simplicity and decrease docker image layers.
Remove unnecessary RUN commands to change file and directory ownership for decrease docker image layers.
Remove unnecessary RUN command to install packages.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -63,18 +30,18 @@ WORKDIR /app/published
...
+              wwwroot/bundles \
+              wwwroot/db_backups \
+              wwwroot/files/exportimport \
+              wwwroot/icons \
+              wwwroot/images \
+              wwwroot/images/thumbs \
+              wwwroot/images/uploaded
 
 # create the runtime instance 
 FROM mcr.microsoft.com/dotnet/aspnet:8.0-alpine AS runtime
```
Comment (danzuep) [inline Dockerfile:47]: Could move this up top, but here's probably more readable.

**Review by danzuep — APPROVED** [review]
This looks way better!

**Comment by skoshelev** [comment]
Hi @ghorbani-ali. Thank you, this is great work.

**Comment by ghorbani-ali** [comment]
> Hi @ghorbani-ali. Thank you, this is great work.

Tanks a lot. it's my pleasure.

**Pull request merged**